src/modules/powerrename/lib/PowerRenameRegEx.cpp

```diff
@@ -201,6 +201,9 @@ HRESULT CPowerRenameRegEx::Replace(_In_ PCWSTR source, _Outptr_ PWSTR* result)
             std::wstring searchTerm(m_searchTerm);
             std::wstring replaceTerm(m_replaceTerm ? wstring(m_replaceTerm) : wstring(L""));
 
+            replaceTerm = regex_replace(replaceTerm, std::wregex(L"\\$[0]+"), L"$");
+            replaceTerm = regex_replace(replaceTerm, std::wregex(L"\\$[0]*[1-9]{2,}"), L"");
```

Comment:
`L"\\$[0]*[1-9][0-9]{1,}"`

Reply:
Yep silly mistake 👍 
To be clear. First one is to handle cases such as `$0001` when there is at least one capturing group. PowerRename doesn't recognize those when there is more than 2 digits.
`[0]*` at the second is kind of redundant but wanted to keep it just in case.